Lection_4/HW_4.py

```diff
@@ -0,0 +1,18 @@
+"""
+1. Створити програму, яка буде очікувати введення тексту від користувача і повідомляти —
+чи є введений текст “числом” чи “словом”.
+
+2. Якщо введений текст “число”, програма має також вказати чи є воно парним чи непарним.
+
+3. Якщо це “слово”, програма має вказати його довжину.
+"""
+
+x = input("Enter please any text: ")
+
+if x.isdigit():
+   if int(x)%2==0:
```

Comment:
It would be better to use spaces between operators to improve code readability: `if int(x) % 2 == 0:`